apps/docker/docker-build-prod.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
 cd /azerothcore
+git config --global --add safe.directory /azerothcore
```

Comment:
Instead of adding this into the docker-build-prod.sh file I would instead add it to the Dockerfile within the base image stage. Probably somewhere around this line: 
https://github.com/azerothcore/azerothcore-wotlk/blob/713cddc1ba534cd29b5d62c3f2fb4ed2626003c9/apps/docker/Dockerfile#L65  

It would need to be with RUN in front of the command.